tests/remove_stale_string.py

```diff
@@ -0,0 +1,19 @@
+import re
+import json
+import glob
+from collections import OrderedDict
+
+locale_folder = "../locales/"
+locale_files = glob.glob(locale_folder + "*.json")
+locale_files = [filename.split("/")[-1] for filename in locale_files]
+locale_files.remove("en.json")
+
+reference = json.loads(open(locale_folder + "en.json").read())
+
+for locale_file in locale_files:
+
+    print(locale_file)
+    this_locale = json.loads(open(locale_folder + locale_file).read(), object_pairs_hook=OrderedDict)
+    this_locale_fixed = {k:v for k, v in this_locale.items() if k in reference}
+
+    json.dump(this_locale_fixed, open(locale_folder + locale_file, "w"), indent=4, ensure_ascii=False)
```

Comment:
```suggestion
    json.dump(this_locale_fixed, open(locale_folder + locale_file, "w"), indent=4, ensure_ascii=False, sort_keys=True)
```

I'm surprised you didn't needed that option but normally you should to avoid breaking order?

Reply:
We can, but the locale files are not ordered in the first place so that would create huge diff that reorders everything :s (Ultimately someday we could try to configure weblate to automatically reorder things when it creates commits I guess)

Reply:
Oh ok, I though they were '^'

I'm really surprised that the order is unchanged then '-'

Reply:
Yup, I had to load the json using an OrderedDict